pmlc/dialect/eltwise/ir/ops.cc

```diff
@@ -142,6 +142,16 @@ DEFINE_CANONICALIZER(TanHOp);
 DEFINE_CANONICALIZER(TanOp);
 DEFINE_CANONICALIZER(SelectOp);
 
+OpFoldResult CastOp::fold(ArrayRef<Attribute> operands) {
+  auto oldType = getRankedTensorType(tensor().getType());
+  auto newType = getRankedTensorType(result().getType());
+  /// cast(x, x.type) -> x
```

Comment:
I don’t understand this comment

Reply:
I was trying to say "in the case where you cast an input tensor `x` to a result type that matches `x`'s initial type, fold `x`"

Any ideas to make the comment more legible? I was just trying to match the comment format that the rest of the folders  were using

Reply:
How about:
```
// cast(x).type == type -> x 
```